### ISSUE TITLE
get_smry, time_index 

### ISSUE THREAD

**Issue by rnyb** [equinor/fmu-ensemble#74]
Make it possible to use a spesific date with get_smry

Example:
date='2019-01-01'
smry = ens.get_smry(column_keys=['G*PTH:*'], time_index=date)

**Comment by berland** [comment]
Current API supports a `list` of `datetime.date` as input for `timeindex`, so this is a workaround for specifying an ISO-formatted string directly:
```
import datetime
ens.get_smry(time_index=[datetime.date(2019, 1, 1)])
```

**Comment by berland** [comment]
Solved in #79
